.github/workflows/fitness-functions.yml

```diff
@@ -17,12 +17,13 @@ jobs:
         with:
           node-version: '16'
 
+      - name: Install dependencies
+        run: yarn
+
       - name: Run fitness functions
         env:
-          HEAD_REF: ${{ github.event.pull_request.head.ref }}
           BASE_REF: ${{ github.event.pull_request.base.ref }}
         run: |
-          # git fetch origin $HEAD_REF
-          # git fetch origin $BASE_REF
-          # git diff origin/$BASE_REF origin/$HEAD_REF -- . > diff
-          # npm run fitness-functions -- "ci" "./diff"
+          git fetch origin $BASE_REF
+          git diff origin/$BASE_REF HEAD -- . > diff
```

Comment:
It looks like `BASE_REF` is expected to be the target branch (e.g. `develop`) at the time this step is run. This seems like the wrong point of comparison because this branch might be missing changes present on `develop` causing the fitness check to pass/fail when it shouldn't.

We had a similar problem when the bundle size check was implemented, and we arrived at this solution: 
https://github.com/MetaMask/metamask-extension/blob/a4a5b28f2e66497bcf5d1a139320322e5c44e075/.circleci/config.yml#L956

To translate to a format that makes sense for GitHub Actions, it'd be something like

```
git diff "$(git merge-base $BASE_REF HEAD)" HEAD -- . > diff
```

This would provide the most recent target branch (e.g. `develop`) commit present on this branch, using that as the comparison commit. This ensures that the check will only pass or fail due to changes in that PR directly.